aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -37,44 +38,38 @@ module aptos_framework::stake {
     friend aptos_framework::genesis;
     friend aptos_framework::reconfiguration;
 
-    /// Lockup period is shorter than required.
-    const ELOCK_TIME_TOO_SHORT: u64 = 1;
-    /// Withdraw not allowed, the stake is still locked.
-    const EWITHDRAW_NOT_ALLOWED: u64 = 2;
     /// Validator Config not published.
-    const EVALIDATOR_CONFIG: u64 = 3;
+    const EVALIDATOR_CONFIG: u64 = 1;
```

Comment:
@gregnazario are you using specific error codes for anything involving staking?
note: this is technically not backwards compatible @movekevin

Reply:
Why is this not backward compatible? Is it because downstream API/tooling might rely on specific error codes? The error map has been moved into the bytecode AFAIK cc @wrwg. So all errors and messages will be updated along with the bytecode. If API/tooling depend on the exact error code, this might change things a bit as we'll need to not reuse deleted error codes.